ci-values.yaml

```diff
@@ -5,9 +5,10 @@ global:
 
 rpe-service-auth-provider:
   java:
-    MICROSERVICEKEYS_CCD_ADMIN: AAAAAAAAAAAAAAAA
-    MICROSERVICEKEYS_CCD_DATA: AAAAAAAAAAAAAAAA
-    MICROSERVICEKEYS_CCD_DEFINITION: AAAAAAAAAAAAAAAA
+    environment:
+      MICROSERVICEKEYS_CCD_ADMIN: AAAAAAAAAAAAAAAA
+      MICROSERVICEKEYS_CCD_DATA: AAAAAAAAAAAAAAAA
+      MICROSERVICEKEYS_CCD_DEFINITION: AAAAAAAAAAAAAAAA
```

Comment:
Is this change required ?

Reply:
syntax was wrong . but it seems we do not need  these values for ci.